docs/src/modules/ROOT/pages/upgrading-timefold-solver/upgrade-from-optaplanner.adoc

```diff
@@ -27,7 +27,7 @@ Gradle::
 --
 [source,shell,subs=attributes+]
 ----
-curl https://timefold.ai/product/upgrade/upgrade-timefold.gradle > upgrade-timefold.gradle ; gradle -Dorg.gradle.jvmargs=-Xmx2G --init-script upgrade-timefold.gradle rewriteRun -DtimefoldSolverVersion={timefold-solver-version} ; rm upgrade-timefold.gradle
+curl https://raw.githubusercontent.com/TimefoldAI/timefold-solver/refs/tags/v{timefold-solver-version}/migration/upgrade-timefold.gradle > upgrade-timefold.gradle ; gradle -Dorg.gradle.jvmargs=-Xmx2G --init-script upgrade-timefold.gradle rewriteRun -DtimefoldSolverVersion={timefold-solver-version} ; rm upgrade-timefold.gradle
```

Comment:
This doesn't work for me. 
example gives 404, because `upgrade-timefold.gradle` doesn't exist? 

https://raw.githubusercontent.com/TimefoldAI/timefold-solver/refs/tags/v1.19.0/migration/upgrade-timefold.gradle

Reply:
Correct, doesn't exist for now. It will when the next release is tagged.